File: in_Python/0153 Find Minimum in Rotated Sorted Array.py
class Solution(object):
    def findMin(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i,j = 0, len(nums)-1
        while i < j:
            m = i + (j-i) / 2
            if nums[m] > nums[j]:
                i = m + 1
            else:
                j = m
        return nums[i]

File: in_Java/0153 Find Minimum in Rotated Sorted Array.java
class Solution {
    public int findMin(int[] nums) {
        int i = 0;
        int j = nums.length-1;
        while(i < j){
            int m = i + (j-i)/2;
            if(nums[m] > nums[j]) i = m + 1;
            else j = m;
        }
        return nums[i];
    }
}